tests/type_inference/test_functions.py

```diff
@@ -4,38 +4,55 @@
 import tests.custom_hypothesis_support as cs
 import hypothesis.strategies as hs
 from typing import Callable
-from python_ta.transforms.type_inference_visitor import register_type_constraints_setter,\
-    environment_transformer, TYPE_CONSTRAINTS
+from python_ta.transforms.type_inference_visitor import TYPE_CONSTRAINTS
 from keyword import iskeyword
 
 
-def _parse_to_function(function_name, args_list, return_value):
+def _parse_to_function(function_name, args_list, return_statement):
     """Helper to parse given data into function definition."""
     return f'def {function_name}({", ".join(args_list)}):' \
-           f'   return {repr(return_value)}'
+           f'   return {repr(return_statement)}'
 
 
-@given(hs.text(alphabet="abcdefghijklmnopqrstuvwxyz", min_size=1), cs.primitive_values)
-def test_function_def_no_args(function_name, return_value):
-    """Test FunctionDef node visitors representing function definitions with no parameters and primitive return val."""
+def _parse_to_function_no_return(function_name, args_list, return_statement):
```

Comment:
Okay, so the problem is that the expression in the function body is explicitly *not* a "return statement" for this function, because that's the whole point of what this function is testing.